client/src/tokens/spacing.js

```diff
@@ -11,6 +11,7 @@ const spacing = {
   4: '1rem',
   5: '1.25rem',
   6: '1.5rem',
+  6.5: '1.625rem',
```

Comment:
This seems to only be used for the height of one element? If so we can hard-code it, there’s no need to add a token that’d be reusable everywhere else.

Reply:
Discussed with Sage: will proceed as-is.